tests/runner.py

```diff
@@ -866,6 +855,18 @@ def setUpClass(self):
     self.harness_port = int(os.environ.get('EMCC_BROWSER_HARNESS_PORT', '9999'))
     if not has_browser():
       return
+
+    if emscripten_browser:
+      cmd = shlex.split(emscripten_browser)
+
+      def run_in_other_browser(url):
+        Popen(cmd + [url])
+
+      webbrowser.open_new = run_in_other_browser
+      print("Using Emscripten browser: " + str(cmd))
+    else:
```

Comment:
nit: this else is so short that it's probably easier to read flipped around

Reply:
Interesting I never considered that.  I think you are probably right.   I guess the argument that you should use positive condition checks is maybe outweighed by the rule that if you have very short case (single line) it should always go first?